Include/refcount.h

```diff
@@ -242,6 +242,18 @@ PyAPI_FUNC(void) Py_DecRef(PyObject *);
 PyAPI_FUNC(void) _Py_IncRef(PyObject *);
 PyAPI_FUNC(void) _Py_DecRef(PyObject *);
 
+#ifndef Py_GIL_DISABLED
```

Comment:
We're planning to unify this for the free-threaded and default build later right? Not much to argue for reduce code divergence when it seems we're still adding more ifdefs :).